mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -336,14 +334,14 @@ def init_base(self):
                 upper_boundary_conditions_particles=[
                     'periodic', 'periodic', 'absorbing'],
                 moving_window_velocity=None,
-                warpx_max_grid_size=self.NZ//self.MAX_GRID_SIZE_FACTOR
+                warpx_max_grid_size=self.NZ//self.MIN_TILES
             )
             '''
 
         self.DT = mwxrun.init_timestep(
             DT=self.DT,
-            V_anode=self.V_ANODE_CATHODE,
             CFL_factor=self.CFL_FACTOR,
+            V_grid=self.V_ANODE_CATHODE
```

Comment:
Without a thermal vmax this should break at flatband if DT is not specified, right? See above comment in the init_timestep code.